app/src/ingestion/pdf_postprocess.py

```diff
@@ -114,8 +114,28 @@ def _add_link_markdown(e_text: EnrichedText) -> EnrichedText:
     return e_text
 
 
+def _add_list_markdown(
+    prev_e_text: EnrichedText | None, current_e_text: EnrichedText
+) -> EnrichedText:
+    if (
+        prev_e_text
+        and prev_e_text.type == TextType.LIST_ITEM
+        and current_e_text.type == TextType.LIST_ITEM
+    ):
+        # if sublist, indent 2 spaces
```

Comment:
https://commonmark.org/help/tutorial/10-nestedLists.html says:
> To nest one list within another, indent each item in the sublist by four spaces. 
https://nava.slack.com/archives/C06DP498D1D/p1725550690832049?thread_ts=1725547115.307169&cid=C06DP498D1D.
```suggestion
        # if sublist, indent 4 spaces
```

Reply:
two spaces also works 
<img width="729" alt="Screenshot 2024-09-06 at 1 19 24 PM" src="https://github.com/user-attachments/assets/e39ae869-0846-4f47-84c5-4aea10f9a51b">

Reply:
👍  [Interesting thread](https://github.com/DavidAnson/markdownlint/issues/137).